NEWS.md

```diff
@@ -881,6 +881,8 @@ Deprecated or removed
   * The functions `eigs` and `svds` have been moved to the `IterativeEigensolvers` standard
     library module ([#24714]).
 
+  * Sparse array functionality has moved to the `SparseArrays` standard library ([#25249]).
```

Comment:
Perhaps "has moved to the `SparseArrays` (package|module) in the standard library", "has moved to the `SparseArrays` standard library (package|module)", or "has moved to `SparseArrays` in the standard library"?